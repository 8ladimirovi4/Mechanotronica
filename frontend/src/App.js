import React from 'react';
import './App.css';
import OverviewFlow from './components/version_1/OverviewFlow';
import AddNodeWhithParentsNodes from './components/version_2/AddNodeWhithParentsNodes';


function App() {
  return ( 
    <>
    {/* <OverviewFlow/> */}
        <AddNodeWhithParentsNodes/> 
        </>
  );
}

export default App;
