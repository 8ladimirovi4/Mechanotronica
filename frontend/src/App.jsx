import React from 'react';
import useResizeObserver from "use-resize-observer/polyfilled";
import './App.css';
import OverviewFlow from './components/version_1/OverviewFlow';
import AddNodeWhithParentsNodes from './components/version_2/AddNodeWhithParentsNodes';


function App() {
  return ( 
    <div>
    {/* <OverviewFlow/> */}
        <AddNodeWhithParentsNodes/> 
        </div>
  );
}

export default App;
