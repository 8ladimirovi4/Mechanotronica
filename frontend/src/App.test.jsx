import { create } from 'react-test-renderer';
import App from './App.jsx';


describe('Components test', () => {
test ('App test', () => {
const component = create(<App/>)
const root = component.root
let a = root.findAllByType('a')
expect(a.length).toBe(1)

});
});