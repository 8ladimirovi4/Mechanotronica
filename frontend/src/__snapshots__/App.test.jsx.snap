// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` testing snapshot components 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;

exports[`AddNodeWhithParentsNodes component testing snapshot components 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;

exports[`AddNodeWhithParentsNodes testing snapshot components 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;

exports[`App component Component created 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;

exports[`App component testin 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;

exports[`App component testing snapshot components 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;

exports[`C testing snapshot components 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;

exports[`foo foo 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;

exports[`foo testing snapshot components 1`] = `
<div
  className="wrapper"
>
  <div
    className="react-flow"
    data-testid="rf__wrapper"
    style={
      Object {
        "height": "100%",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
        "zIndex": 0,
      }
    }
  >
    <div
      className="react-flow__renderer"
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
        }
      }
    >
      <div
        className="react-flow__pane"
        onClick={[Function]}
        onContextMenu={[Function]}
        onWheel={[Function]}
        style={
          Object {
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      >
        <div
          className="react-flow__viewport react-flow__container"
          style={
            Object {
              "transform": "translate(0px,0px) scale(1)",
            }
          }
        >
          <div
            className="react-flow__edgelabel-renderer"
          />
          <div
            className="react-flow__nodes"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="react-flow__panel react-flow__attribution bottom right"
      data-message="Please only hide this attribution when you are subscribed to React Flow Pro: https://pro.reactflow.dev"
      style={
        Object {
          "pointerEvents": "all",
        }
      }
    >
      <a
        aria-label="React Flow attribution"
        href="https://reactflow.dev"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Flow
      </a>
    </div>
    <div
      id="react-flow__node-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select a node.
      You can then use the arrow keys to move the node around.
       Press delete to remove it and escape to cancel.
       
    </div>
    <div
      id="react-flow__edge-desc-1"
      style={
        Object {
          "display": "none",
        }
      }
    >
      Press enter or space to select an edge. You can then press delete to remove it or escape to cancel.
    </div>
    <div
      aria-atomic="true"
      aria-live="assertive"
      id="react-flow__aria-live-1"
      style={
        Object {
          "border": 0,
          "clip": "rect(0px, 0px, 0px, 0px)",
          "clipPath": "inset(100%)",
          "height": 1,
          "margin": -1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": 1,
        }
      }
    />
  </div>
</div>
`;
